spec/system/visiting_the_topics_view_page_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe 'visiting topics page' do 
+  scenario 'when viewing' do
+  	visit('/topics')
```

Comment:
Personally I prefer using route helpers to save updating the tests when/if you change the routes, but what you've done is fine too (perhaps better 🤔 ).
Using route helpers it would look like:
```
visit topics_path
```